packages/core/src/app/shipping/Shipping.tsx

```diff
@@ -434,6 +437,7 @@ export function mapToShippingProps({
         updateShippingAddress: checkoutService.updateShippingAddress,
         isFloatingLabelEnabled: isFloatingLabelEnabled(config.checkoutSettings),
         shouldRenderStripeForm: providerWithCustomCheckout === PaymentMethodId.StripeUPE && shouldUseStripeLinkByMinimumAmount(cart),
+        isDebouncedCheckAndUpdateAddressExperiment: features['CHECKOUT-8006.move_check_updated_address_value_within_debounced'] ?? false,
```

Comment:
For better supporting expeirment removal, we now use this method to get the value.
https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/app/common/utility/isExperimentEnabled.ts